packages/rollup/src/package.json

```diff
@@ -22,5 +25,8 @@
         "npm_bazel_rollup": {
             "rootPath": "."
         }
+    },
+    "dependencies": {
+        "@bazel/worker": "0.0.0-PLACEHOLDER"
```

Comment:
Should we make this a `peerDependency` or `optionaldependencies` since it is opt-in atm?